pcsx2/MemoryCardFolder.cpp

```diff
@@ -2238,6 +2238,12 @@ bool FileAccessHelper::CleanMemcardFilenameEndDotOrSpace(char* name, size_t leng
 
 bool MemoryCardFileMetadataReference::GetPath(std::string* fileName) const
 {
+
```

Comment:
You can remove the extra line.

Reply:
Will do, wasn't sure about the line spacings lol